client/src/utils/fetch.js

```diff
@@ -19,7 +19,7 @@ const baseRequest = {
   credentials: 'include',
 };
 
-async function _getAPI(uri) {
+const _getAPI = async uri => {
   const { API_URL } = getConfig();
```

Comment:
question: Why is getConfig a function instead of having config.js directly supply the config object?

Reply:
Answer to most comments, including this one: This PR reverts to the state that `buyProduct` and `checkout` were as at https://github.com/GoogleCloudPlatform/avocano/blob/8afc2b99fd0be3c3e4a37277ef4447b9b7a38065/client/src/utils/fetch.js. This PR is designed to revert the changes back to this known "working" state. Not "good" state, as there's still issues with it (as noted with all your comments), but it's a functional state.

Reply:
https://github.com/GoogleCloudPlatform/avocano/issues/339